docs/config/README.md

```diff
@@ -841,20 +841,20 @@ it would have been `nixpkgs/pkgs`.
+| Option              | Default                                                                                                     | Description                                                                                                                        |
+| ------------------- | ----------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------- |
+| `truncation_length` | `3`                                                                                                         | The number of parent folders that the current directory should be truncated to.                                                    |
+| `truncate_to_repo`  | `true`                                                                                                      | Whether or not to truncate to the root of the git repo that you're currently in.                                                   |
+| `format`            | `"[$path]($style)[$read_only]($read_only_style) "`                                                          | The format for the module.                                                                                                         |
+| `style`             | `"bold cyan"`                                                                                               | The style for the module.                                                                                                          |
+| `disabled`          | `false`                                                                                                     | Disables the `directory` module.                                                                                                   |
+| `read_only`         | `"🔒"`                                                                                                       | The symbol indicating current directory is read only.                                                                              |
+| `read_only_style`   | `"red"`                                                                                                     | The style for the read only symbol.                                                                                                |
+| `truncation_symbol` | `""`                                                                                                        | The symbol to prefix to truncated paths. eg: "…/"                                                                                  |
+| `repo_root_style`   | `None`                                                                                                      | The style for the root of the git repo when `truncate_to_repo` option is set to false. The default value is equivalent to `style`. |
```

Comment:
Please revert this. It also works with `truncate_to_repo` enabled.

Reply:
I reverted README.md